portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,9 +29,32 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  /**
+  * Holds the stubbed comments to be returned by the GET method.
+  */
+  private final String[] COMMENTS = new String[]{
```

Comment:
Maybe you could initialize this directly using `Arrays.asList`?  Although, if you really want to follow the guideline, you apparently need to use `ImmutableList<String> COMMENTS = ImmutableList.of("Comment one", ...)` (https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names)

Reply:
So I looked into this, and it seems the constant rules only apply if the field is static. It doesn't make sense for this to be a static field, so I updated the field to follow rules for a private final variable.

Thanks for pointing this out! I'm going to study the style guide more closely tomorrow

Reply:
If the list is going to change, then indeed you don't need to use `ImmutableList`. Though, having a final reference would still be desirable I think (see [this](https://softwareengineering.stackexchange.com/a/98703) for more info).

Reply:
I generally agree with making everything as immutable as possible - both for references and the values they hold. I would have probably used ImmutableList in this PR because you never actually change the value in this version. (No need to change this now - just a drive-by comment.)

For production code, I like to follow the principle of, "code is read (by other people) much more often that it is changed". Meaning that you should choose to make code easier to understand at the expense of making it harder to modify.

(This is a private variable, so changing its type is not going to require changes outside this file. The argument would be different for a public property that is part of the class API.)